crates/rust-client/src/errors.rs

```diff
@@ -92,6 +92,8 @@ pub enum ClientError {
     TransactionScriptError(#[source] TransactionScriptError),
     #[error("client initialization error: {0}")]
     ClientInitializationError(String),
+    #[error("account with id {0} not found on the network")]
+    AccountNotFoundOnNetwork(AccountId),
```

Comment:
I would change this to:
```suggestion
    #[error("account with id {0} not found on chain")]
    AccountNotFoundOnChain(AccountId),
```

So that it's in line with the other variant `NoteNotFoundOnChain`